exercises/collatz-conjecture/collatz_conjecture_test.exs

```diff
@@ -0,0 +1,48 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("collatz_conjecture.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule CollatzConjectureTest do
+  use ExUnit.Case
+
+  test "zero steps for one" do
+    assert CollatzConjecture.calc(1) == 0
+  end
+
+  test "zero is an error" do
+    assert_raise ArgumentError, "Only positive numbers are allowed", fn -> CollatzConjecture.calc(0) end
```

Comment:
Yeah, we don't _have_ to follow the pattern exactly. Different languages have different ways of expressing error cases. This is honestly one of the hardest parts of this track, since there are totally valid arguments for each option (raising an exception vs. returning an error tuple).

I've been using this blog post as a guideline, which might be helpful to consult here: http://michal.muskala.eu/2017/02/10/error-handling-in-elixir-libraries.html#errors-vs-exceptions